source/user-manual/capabilities/policy-monitoring/configuration_assessment/index.rst

```diff
@@ -0,0 +1,15 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _policy_monitoring_introduction:
+
+Configuration assessment
+========================
+
+Wazuh has improved its policy monitoring process with a new module that include the functionality that *Rootcheck* provided. The policy files are now updated and translated to YAML with more interesting fields, this files are parsed by this new module and after that the scans are performed, in each of them, the new information gathered is stored at a database where differences between consecutive scans are made, this makes possible generating an alert for every check that has changed its result. Also, an statistical alert that summaries the checking process is generated in every scan.
```

Comment:
an statistical -> a statistical
include -> includes the functionality
this files -> these files